tawazi/errors.py

```diff
@@ -1,11 +1,12 @@
-from enum import Enum
+from enum import Enum, unique
 
 
 class DAGBaseException(BaseException):
     pass
 
 
-class ErrorStrategy(Enum):
+@unique
```

Comment:
question : what does it do ?

Reply:
This makes sure that each `value` related to each `memember` of the enum is unique: [unique](https://docs.python.org/3/library/enum.html#:~:text=their%20Flag%20membership.-,enum.unique(),-Enum%20class%20decorator)